admin-core/assets/src/dashboard-app/pages/welcome/Welcome.js

```diff
@@ -179,6 +179,95 @@ const Welcome = () => {
 								</div>
 							</div>
 						</section>
+						<section aria-labelledby="section-2-title">
+							<h2 className="sr-only" id="section-2-title">
+								Section title
+							</h2>
+							<div className="box-border rounded-md bg-white shadow-sm overflow-hidden transition hover:shadow-hover">
+								<div className="p-8 pr-10">
+									<h3 className="text-slate-800 text-xl font-medium pb-2">
+										{ __( 'Rate Us', 'ultimate-addons-for-gutenberg' ) }
+									</h3>
```

Comment:
We actually don't need these section titles, since the title "Rate Us" is already in the accessible format.

Besides that, the ID `section-2-title` seems to be in a lot of elements, which is actually not idea.

Let's do this instead.
```suggestion
						<section>
							<div className="box-border rounded-md bg-white shadow-sm overflow-hidden transition hover:shadow-hover">
								<div className="p-8 pr-10">
									<h3 className="text-slate-800 text-xl font-medium pb-2">
										{ __( 'Rate Us', 'ultimate-addons-for-gutenberg' ) }
									</h3>
```

Reply:
@bsf-zanev Done with the changes